MMM-pages.js

```diff
@@ -185,31 +185,51 @@ Module.register('MMM-pages', {
    * Animates the page change from the previous page to the current one. This
    * assumes that there is a discrepancy between the page currently being shown
    * and the page that is meant to be shown.
+   * 
+   * @param {string} [name] - the name of the hiddenPage we want to show. Optional and only used when we want to switch to a hidden page
    */
-  animatePageChange: function () {
+  animatePageChange: function (name) {
```

Comment:
Do we need to? In most of the cases this is still used for switching to another page index instead of a named page. In that case `targetPageName` does not seem to fit. I feel `animatedPageChange` describes exactly what this functions main task is :)

Reply:
Oh, I mean the variable `name`, not `animatedPageChange`.

Reply:
forgot to fix this. I'll do it tomorrow.